cwms_radar_api/src/test/java/cwms/radar/data/dto/TimeSeriesTest.java

```diff
@@ -0,0 +1,81 @@
+package cwms.radar.data.dto;
+
+import java.time.Duration;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import io.javalin.plugin.json.JavalinJackson;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TimeSeriesTest
+{
+
+	@Test
+	public void testRoundtripJson() throws JsonProcessingException
+	{
+		String tsId = "RYAN3.Stage.Inst.5Minutes.0.ZSTORE_TS_TEST";
+
+		ZonedDateTime start = ZonedDateTime.parse("2021-06-21T14:00:00-07:00[PST8PDT]");
+		ZonedDateTime end = ZonedDateTime.parse("2021-06-22T14:00:00-07:00[PST8PDT]");
+		TimeSeries ts = new TimeSeries(null, -1, 0, tsId, "LRL", start, end, null, Duration.ZERO);
+
+		ObjectMapper om = buildObjectMapper();
+
+		String tsBody = om.writeValueAsString(ts);
+		assertNotNull(tsBody);
+
+		TimeSeries ts2 = om.readValue(tsBody, TimeSeries.class);
```

Comment:
The test fails on this line, with the following message:
com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException at TimeSeriesTest.java:37

Reply:
Oh, I fixed that too. I was trying to figure out why my IDE was listing TimeSeries as changed in two changelists.  Weird.  I'll push that too.

Reply:
If my IDE complains about unused imports I usually just let it "optimize imports" or whatever it calls it.  I don't pay too much attention to how imports are ordered.